packages/@angular/cli/commands/config.ts

```diff
@@ -0,0 +1,135 @@
+import { Command } from '../models/command';
+import * as fs from 'fs';
+import { CliConfig } from '../models/config';
+import { oneLine } from 'common-tags';
+
+const SilentError = require('silent-error');
+
+
+export interface ConfigOptions {
+  jsonPath: string;
+  value?: string;
+  global?: boolean;
+}
+
+export default class ConfigCommand extends Command {
+  public readonly name = 'config';
+  public readonly description = 'Get/set configuration values.';
+  public readonly arguments = ['jsonPath', 'value'];
+  public readonly options = [
+    {
+      name: 'global',
+      type: Boolean,
+      'default': false,
+      aliases: ['g'],
+      description: 'Get/set the value in the global configuration (in your home directory).'
+    }
+  ];
+
+  public run(options: ConfigOptions) {
+    const config = options.global ? CliConfig.fromGlobal() : CliConfig.fromProject();
+
+    if (config === null) {
+      throw new SilentError('No config found. If you want to use global configuration, '
+        + 'you need the --global argument.');
+    }
+
+    const action = !!options.value ? 'set' : 'get';
+
+    if (action === 'get') {
+      return this.get(config, options);
+    } else {
+      return this.set(config, options);
+    }
+  }
+
+  private get(config: CliConfig, options: ConfigOptions) {
+    const value = config.get(options.jsonPath);
+
+    if (value === null || value === undefined) {
+      throw new SilentError('Value cannot be found.');
+    } else if (typeof value == 'object') {
+      this.logger.info(JSON.stringify(value, null, 2));
+    } else {
+      this.logger.info(value.toString());
+    }
+  }
+
+  private set(config: CliConfig, options: ConfigOptions) {
+    const type = config.typeOf(options.jsonPath);
+      let value: any = options.value;
+      switch (type) {
+        case 'boolean': value = this.asBoolean(options.value); break;
+        case 'number': value = this.asNumber(options.value); break;
+        case 'string': value = options.value; break;
+
+        default: value = this.parseValue(options.value, options.jsonPath);
+      }
+
+      if (options.jsonPath.indexOf('prefix') > 0) {
```

Comment:
We probably should strengthen this check at some point as it will catch anything with the word `prefix`.